src/processTargets/modifiers/ModifyIfImplicitScopeTypeStage.ts

```diff
@@ -0,0 +1,37 @@
+import { Target } from "../../typings/target.types";
+import { ModifyIfImplicitScopeTypeModifier } from "../../typings/targetDescriptor.types";
+import { ProcessedTargetsContext } from "../../typings/Types";
+import getModifierStage from "../getModifierStage";
+import { ModifierStage } from "../PipelineStages.types";
+
+/**
+ * Runs {@link ModifyIfImplicitScopeTypeModifier.modifier} if the target is lacking an explicit modifier.
+ */
+export default class ModifyIfImplicitScopeTypeStage implements ModifierStage {
```

Comment:
I don't love the term "implicit scope type", because that implies it has a scope type. I think the clearest would be `ModifyIfNoExplicitScopeTypeStage`, but that's starting to get a bit verbose even for me, esp when you get to `containingSurroundingPairIfNoExplicitScopeTypeStage` 😅. So maybe we go with something like `ModifyIfUntypedStage`, and then eg `containingSurroundingPairIfUntypedStage`?